docs/SOP.md

```diff
@@ -30,11 +30,12 @@ In general, the metadata record of an ontology in the OBO Foundry metadata regis
 
 1. Any member of the community (OBO Foundry or otherwise) may propose a change in the form of a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests).
 2. This pull request is reviewed by a member of the OBO Foundry operations committee and may be rejected.
-3. The OBO Foundry operations committee member MUST then tag the listed contact person using their GitHub handle on the pull request requesting a review (initial request for feedback).
-4. After two weeks without activity, an OBO Foundry operations committee member will update the pull request and announce a prospective merge date no earlier than two weeks in the future.
-5. After at least 4 weeks have passed since the initial request for review:
+3. If the proposed change affects merely the _presentation_ of the web page (information moved from one section of the website to another; changed URLs from http to https, change in the formatting (dimensions) of the ontology logo, typos, the OBO Foundry operations committee member may choose to accept the pull request without waiting for further review (Note, this rule is important to not block website development for months at a time). If the previous case exists, the following points may be ignored. 
```

Comment:
This line is the only line that changed, @lschriml, @nataled, the other ones only appear changed due to the change in numbering.